stograde/formatters/base.py

```diff
@@ -1,8 +1,16 @@
 from collections import defaultdict
-from stograde.common import group_by as group
+from typing import List, TYPE_CHECKING
 
+from ..common import group_by as group
```

Comment:
```suggestion
from ..common.group_by import group_by as group
```

then we can change the lines below back to `group(`?

Reply:
I was having trouble with it earlier but it seems to be working fine now